src/components/button/Button.tsx

```diff
@@ -1,21 +1,24 @@
 import * as React from 'react';
-import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent, Platform, AccessibilityTrait, ViewStyle } from 'react-native';
-
-const s = require('./Button.css');
+import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent,
+  Platform, AccessibilityTrait, ViewStyle, TextStyle } from 'react-native';
```

Comment:
I don't know if it's available with default tsconfig, otherwise, we could write down our own rule, but something to make the blank line mandatory between a block of imports would be nice. At least, it's my personal feedback.

My personal rules are:

- global dependencies
- utils/stores/config
- global components
- local components/local style import (if import used)
- interface
- const

Reply:
We can use the `tslint-origin-ordered-imports-rule` tslint plugin.

`"origin-ordered-imports": [true, "one-blank-line"]`

And it would look like this:

```tsx
import * as React from 'react';
import { View, Text } from 'react-native';
import { observer } from 'mobx-react';
import { Navigation } from 'react-native-navigation';
import { CodePush } from 'react-native-code-push';

import { UI } from '~stores/UI';
import { Counter } from '~stores/Counter';
import { COUNTER } from '~screens';
import { codePushConfig } from '~utils/code-push';
import { Button } from '~components/button';

import s from './Home.css';
```

Sold!

Reply:
I would also add something to separate `~/components` from `~stores/utils/screens` to make it more clear between `DOM` stuff and `data/functions`